examples/calculate_rice_score.py

```diff
@@ -0,0 +1,146 @@
+# Reference: https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/
+from python_http_client import Client
+import os
+import json
+from datetime import datetime
+from email.utils import parsedate_tz, mktime_tz
+
+client = Client(host="http://{}".format(os.environ.get('DX_IP')))
+
+def get_repo_name(url):
+    split = url.rsplit('/',3)
+    return split[1]
+
+def calculate_rice_reach(task):
+    # Last updated May 2019
+    # https://docs.google.com/spreadsheets/d/1wNImudaEewPijd9EcgsHiEXbermh4amyug-2tTfyOWQ/edit#gid=2047355119
+    rice_reach = os.getenv('DEFAULT_REACH')
+    if 'sendgrid-python' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_PYTHON_REACH')
+    if 'python-http-client' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_PYTHON_REACH')
+    if 'sendgrid-php' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_PHP_REACH')
+    if 'php-http-client' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_PHP_REACH')
+    if 'sendgrid-csharp' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_CSHARP_REACH')
+    if 'csharp-http-client' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_CSHARP_REACH')
+    if 'sendgrid-java' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_JAVA_REACH')
+    if 'java-http-client' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_JAVA_REACH')
+    if 'sendgrid-nodejs' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_NODEJS_REACH')
+    if 'nodejs-http-client' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_NODEJS_REACH')
+    if 'sendgrid-ruby' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_RUBY_REACH')
+    if 'ruby-http-client' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_RUBY_REACH')
+    if 'sendgrid-go' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_GO_REACH')
+    if 'rest' == get_repo_name(task['url']):
+        rice_reach = os.getenv('TWILIO_SENDGRID_GO_REACH')
```

Comment:
This is a lot of repetition. Prefer to use dictionary lookup like:
```
repo_to_reach_env_var = {
  'sendgrid-python': 'TWILIO_SENDGRID_PYTHON_REACH'
  ...
}
reach_env_var = repo_to_reach_env_var.get(get_repo_name(task['url']), 'MAXIMUM_REACH')
rice_reach = os.getenv(reach_env_var)
```

Reply:
Nice!